scripts/sysman/device.yml

```diff
@@ -355,6 +355,66 @@ returns:
     - $X_RESULT_ERROR_INVALID_SIZE:
         - "The provided value of pCount is not big enough to store information about all the processes currently attached to the device."
 
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Device Description"
+version: "1.9"
+class: $sDevice
+name: $s_device_description_t
```

Comment:
these should be just properties.

Reply:
we need to come up with better name here as we already have zes_device_properties

Reply:
@aravindksg @jandres742 new name such as zes_subdevice_properties_t{stype, pnext, onSubdevice, subDeviceId uuid}?, this new struct could be used for both new APIs.

Reply:
maybe subdevice_properties is good. @aravindksg what do you think?

Reply:
yes,  that should work. if we are going with `zeDeviceGetSubDeviceProperties` as API name then this keeps it in sync

Reply:
Done.